### PR TITLE
update style

### PR DESCRIPTION
**Pull request by noavrd**
update style according to this ticket: https://doitintl.atlassian.net/jira/software/c/projects/CMP/issues/CMP-9504?filter=myopenissues

### CODE REVIEW

File: default.css [hunk 1]
```diff
@@ -131,22 +419,187 @@ table.table thead .sorting_desc_disabled {
...
+}
+
+.table-with-filters {
+  display: flex;
+  flex-direction: column;
+  padding-top: 64px;
+}
+
+.table {
+  width: 100% !important;
```
Comment (hemlata-pathak) [inline default.css:487]: I think it's duplicate. There is already some styles for [table](https://github.com/doitintl/gcpinstances.info/pull/38/files#diff-bec084bdca5e9e44aaac6baa3a0dfb80524284f7e187b7b3f4d7fd667879724eR394) here

File: default.js [hunk 2]
```diff
@@ -26,29 +26,54 @@ function init_data_table() {
...
+    bInfo: false,
+    bStateSave: true,
+    orderCellsTop: true,
+    oSearch: {
+      bRegex: true,
+      bSmart: false,
     },
-    "aoColumnDefs": [
+    columns: [
+      { className: "" },
```
Comment (hemlata-pathak) [inline default.js:52]: you can also give null here. https://datatables.net/reference/option/columns.className

Comment (noavrd) [reply default.js]: @hemlata-pathak fixed

File: default.css (hunk 1 above)
Comment (noavrd) [reply default.css]: @hemlata-pathak fixed

**Review by hemlata-pathak — APPROVED** [review]

**Pull request merged**